_data/projects.yml

```diff
@@ -1,14 +1,9 @@
-- title: Online Shopping App
```

Comment:
Is there a reason you changed the list of projects? This is problematic because this will conflict with other changes and it seems like an unnecessary change.